compose/gettingstarted.md

```diff
@@ -98,73 +96,142 @@ This Compose file defines two services, `web` and `redis`. The web service:
+    ```
+
+    Compose pulls a Redis image, builds an image for your code, and start the
+    services you defined. In this case, the code is statically copied into the image at build time.
+
+2.  Enter `http://0.0.0.0:5000/` in a browser to see the application running.
+
+    If you're using Docker on Linux natively, then the web app should
+    now be listening on port 5000 on your Docker daemon host.
+    If `http://0.0.0.0:5000` doesn't resolve, you can also
+    try `http://localhost:5000`.
+
+    If you're using Docker Machine on a Mac, use  
+    `docker-machine ip MACHINE_VM` to get the IP address of your
+    Docker host. Then, open `http://MACHINE_VM_IP:5000` in a browser.
+
+    You should see a message in your browser saying:
+    ```
+    Hello World! I have been seen 1 times.
+    ```
+
+3.  Refresh the page.
+
+    The number should increment.
+
+    ```
+    Hello World! I have been seen 2 times.
+    ```
 
-        $ docker-compose up
-        Pulling image redis...
-        Building web...
-        Starting composetest_redis_1...
-        Starting composetest_web_1...
-        redis_1 | [8] 02 Jan 18:43:35.576 # Server started, Redis version 2.8.3
-        web_1   |  * Running on http://0.0.0.0:5000/
-        web_1   |  * Restarting with stat
+4.  Switch to another terminal window, and type `docker image ls` to
+    list local images.
 
-   Compose pulls a Redis image, builds an image for your code, and start the
-   services you defined.
+    Listing images at this point should return `redis` and `web`.
 
-2. Enter `http://0.0.0.0:5000/` in a browser to see the application running.
+    ```
+    $ docker image ls
+    REPOSITORY              TAG                 IMAGE ID            CREATED             SIZE
+    composetest_web   latest              e2c21aa48cc1        4 minutes ago       93.8MB
+    python                  3.4-alpine          84e6077c7ab6        7 days ago          82.5MB
+    redis                   alpine              9d8fa9aa0e5b        3 weeks ago         27.5MB
+    ```
 
-   If you're using Docker on Linux natively, then the web app should now be
-   listening on port 5000 on your Docker daemon host. If `http://0.0.0.0:5000`
-   doesn't resolve, you can also try `http://localhost:5000`.
+    You can inspect images with `docker inspect <tag or id>`.
 
-   If you're using Docker Machine on a Mac, use `docker-machine ip MACHINE_VM` to get
-   the IP address of your Docker host. Then, open `http://MACHINE_VM_IP:5000` in a
-   browser.
+5.  Stop the application, either by running `docker-compose down` from
+within your project directory in the second terminal, or CTRL+C in the
+original terminal where you started the app.
 
-   You should see a message in your browser saying:
+## Step 5: Edit the Compose file to add a host mount as a volume
 
-   `Hello World! I have been seen 1 times.`
+Edit `docker-compose.yml` in your project directory to add a volume mount for the `web` service:
```

Comment:
For consistency instead of "volume mount", consider using "bind mount".
@mstanleyjones created https://docs.docker.com/engine/admin/volumes/ that makes clear the difference between bind mounts and volumes. If we use the terms consistently people will understand these concepts faster.